sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md

```diff
@@ -1,74 +1,78 @@
 # OAuth 2.0 Sample for Azure AD Spring Boot Starter client library for Java
 
 ## Key concepts
-This sample illustrates how to use `azure-spring-boot-starter-active-directory` package to work with OAuth 2.0 and OpenID Connect protocols on Auzre. This sample will use Microsoft Graph API to retrieve user infomation
+This sample illustrates how to use `azure-spring-boot-starter-active-directory` package to work with OAuth 2.0 and OpenID Connect protocols on Auzre. This sample will use Microsoft Graph API to retrieve user infomation.
 
 ## Getting started
-
 ### Environment checklist
 We need to ensure that this [environment checklist][ready-to-run-checklist] is completed before the run.
 
-### Register your application with your Azure Active Directory Tenant
-
-Follow the guide [here](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app).
-### Platform configurations
-
-![Platform configurations](docs/image-platform-configurations.png "Platform configurations")
-
-- Set redirect URIs with `http://localhost:8080/login/oauth2/code/azure`.
-- Ensure **Access tokens** and **ID tokens** are selected.
-- To use on-demand authorization of certain resource, you need to add redirect URIs of `http://localhost:8080/login/oauth2/code/{registration-id}`. In this sample, set redirect URIs with `http://localhost:8080/login/oauth2/code/arm`.
-### Configure necessary API permissions
-The sample retrieves user's group memberships using Microsoft graph API which requires the registered app to have `Directory.AccessAsUser.All` permission under `Delegated Permissions` of `Microsoft Graph`, which can allow an application to access the directory as the signed-in user. Also, to display the function of calling multiple resources, this sample will acquire `ActivityFeed.Read` permission under `Office 365 Management APIs` resource. You need AAD admin privilege to be able to grant the permission in API ACCESS -> Required permission. You can follow the below steps:	
-
-* In the list of pages for the app, select **API permissions**	
-   - Click the **Add a permission** button	
-   - Ensure that the **Microsoft APIs** tab is selected	
-   - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**	
-   - In the **Delegated permissions** section, ensure that the right permissions are checked: **Directory.AccessAsUser.All**	
-   - Select the **Add permissions** button
-   - Under **Office 365 Management APIs** tab
-   - Select **Delegated permissions**, and then click **ActivityFeed.Read**
-   - Select the **Add permissions** button
-   - Click **Grant Permissions...** and Yes when prompted.
+### Configure web api
```

Comment:
We do not need to use `web api` in this sample.

Reply:
If we need sample that include both webapp and web api, we can create another sample.